src/openrct2/world/TileElement.h

```diff
@@ -434,9 +434,10 @@ struct SmallSceneryElement : TileElementBase
     ObjectEntryIndex entryIndex; // 5
     uint8_t age;                 // 7
     uint8_t Colour[3];           // 8
+    uint8_t Flags2;              // B
```

Comment:
We can probably remove the comment for the offsets, also I don't see this flag being exported is that done on purpose?

Reply:
Argh

Reply:
Just checked and its actually all okay in terms of export. We assume PoD for the tile elements. As for the offsets they are still somewhat useful.

Reply:
I see, I just checked and you are right.